### PR TITLE
Standardize and update code

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: htdocs/core/class/html.formsms.class.php [hunk 1]
```diff
@@ -88,24 +88,22 @@ function __construct($db)
...
 
-    // phpcs:disable PEAR.NamingConventions.ValidFunctionName.NotCamelCaps
     /**
      *	Show the form to input an sms.
      *
      *	@param	string	$morecss        Class on first column td
      *  @param  int     $showform       Show form tags and submit button (recommanded is to use with value 0)
      *	@return	void
      */
+    // phpcs:ignore PEAR.NamingConventions.ValidFunctionName.NotCamelCaps
```
Comment (frederic34) [inline htdocs/core/class/html.formsms.class.php]: do not change, you will get an error "missing doc comment"

Comment (grandoc) [reply htdocs/core/class/html.formsms.class.php]: I didn't touch it ?

Comment (frederic34) [reply htdocs/core/class/html.formsms.class.php]: github is going crazy sometimes...

Comment (grandoc) [reply htdocs/core/class/html.formsms.class.php]: what can I do ? The fault was not of my own and Frederic correct it...

Comment (frederic34) [reply htdocs/core/class/html.formsms.class.php]: the working code is:
    // phpcs:disable PEAR.NamingConventions.ValidFunctionName.NotCamelCaps
    /**
     *	Show the form to input an sms.
     *
     *	@param	string	$morecss        Class on first column td
     *  @param  int     $showform       Show form tags and submit button (recommanded is to use with value 0)
     *	@return	void
     */
    function show_form($morecss='titlefield', $showform=1)
    {
        // phpcs:enable
        global $conf, $langs, $user, $form;
        if (! is_object($form)) $form=new Form($this->db);

**Comment by frederic34** [comment]
for the space...
https://github.com/Dolibarr/dolibarr/pull/9517

**Pull request merged**